docs/DeveloperGuide.md

```diff
@@ -1824,19 +1830,30 @@ otherwise)
 4. A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be
    able to accomplish most of the tasks faster using commands than using the mouse.
 5. Data stored should be persistent until removal by user, and Private Contact Details should be secure.
-6. The project is expected to adhere to a schedule which delivers a feature set every milestone up to _V1.3_
-7. The application is not expected to
+6. Data should be stored locally.
+7. The GUI should not cause any resolution-related inconveniences to the user for standard screen resolutions of
+   1920x1080 and higher, and should be usable for screen resolutions of 1280x720 and higher.
+8. The application should be packaged into a single JAR file with size not exceeding 100MB.
+9. The project is expected to adhere to a schedule which delivers a feature set every milestone up to _V1.3_
+10. The application is not expected to
     1. Perform Inventory Management
     2. Perform Route Planning
 
 _{More to be added}_
 
 ### Glossary
 
-- **Mainstream OS**: Windows, Linux, Unix, OS-X
-- **Private Contact Detail**: A contact detail that is not meant to be shared with others
-- **CLI**: Command Line Interface
-- **Owner**: The customer who owns the home-based business and who uses the app
+| Term                           | Definition                                                                              |
+|--------------------------------|-----------------------------------------------------------------------------------------|
+| Alphanumeric                   | Consisting of only letters and numbers                                                  |
+| Command Line Interface (CLI)   | A text-based user interface used to run programs                                        |
+| Graphical User Interface (GUI) | A visual interface where you can interact with the program through graphical components |
+| JSON                           | Short for JavaScript Object Notation, a lightweight format for storing your data        |
+| Owner                          | The customer who owns the home-based business and who uses the app                      |
```

Comment:
Should the definition be "The individual who owns the home-based business and who uses the HomeBoss app."

Reply:
Updated!